src/components/Personalization/index.js

```diff
@@ -17,13 +17,57 @@ import { hideContainers, showContainers } from "./flicker";
 import collectClicks from "./helper/clicks/collectClicks";
 
 const DECISIONS_HANDLE = "personalization:decisions";
-
+const ALL_SCOPES = "all_scopes";
 // This is used for Target VEC integration
 const isAuthoringMode = () => document.location.href.indexOf("mboxEdit") !== -1;
 const mergeMeta = (event, meta) => {
   event.mergeMeta({ personalization: { ...meta } });
 };
 
+const mergeQuery = (event, details) => {
+  event.mergeQuery({ personalization: { ...details } });
+};
+
+const storeDecisions = (storage, decisions) => {
+  if (!decisions) {
+    return;
+  }
+
+  const filteredDecisions = {};
+
+  decisions.forEach(decision => {
+    const key = decision.scope || ALL_SCOPES;
```

Comment:
This isn't really all of the scopes.  Would DEFAULT_SCOPE make more sense?

Reply:
actually it is neither all scopes neither default, it is a page wide scope,  maybe PAGE_SCOPE?